src/database/standalone-sqlite.ts

```diff
@@ -2329,13 +2337,25 @@ export class StandaloneSqliteDatabaseWorker {
     dataOffset: number;
     dataSize: number;
   }) {
+    const currentTimestamp = currentUnixTimestamp();
+    const dataId = fromB64Url(id);
+
     this.stmts.data.insertNestedDataId.run({
-      id: fromB64Url(id),
+      id: dataId,
       parent_id: fromB64Url(parentId),
       data_offset: dataOffset,
       data_size: dataSize,
-      indexed_at: currentUnixTimestamp(),
+      indexed_at: currentTimestamp,
     });
+
+    const parentAttributes = this.getDataAttributes(parentId);
```

Comment:
@djwhitt Is this the best way to check if parent is verified? If so I think we should create a cache for it as this query is expensive to run for each data item of the same parent

Reply:
check on update query